site/releases/_how-to-upgrade.qmd

```diff
@@ -11,16 +11,23 @@ To access the latest version of the {{< var validmind.platform >}},^[[Log in to
 
 #### {{< var validmind.developer >}}
 
-To upgrade the {{< var validmind.developer >}}:^[[Get started with the {{< var validmind.developer >}}](/developer/get-started-developer-framework.qmd)]
+To upgrade to the latest version of the {{< var validmind.developer >}}:^[[Get started with the {{< var validmind.developer >}}](/developer/get-started-developer-framework.qmd)]
 
-- [Using JupyterHub](/get-started/developer/try-with-jupyterhub.qmd): Hard refresh your browser tab and re-run the `%pip install --upgrade validmind` cell.
+1. Within your Jupyter Notebook:
```

Comment:
```suggestion
1. In your Jupyter Notebook:
```

(If a smaller word would work rather than a bigger one, always use the smaller one.)